tools/deserialize-proto/main.go

```diff
@@ -55,16 +54,14 @@ func main() {
 				text, err)
 		}
 
-		message := messageType
-		if err := unmarshalProto(message, b); err != nil {
+		if err := proto.Unmarshal(b, msg); err != nil {
 			log.Fatalf("error unmarshaling proto, text = %s, err = %v", text, err)
 		}
 
-		pjson, err := json.MarshalIndent(message, "", "  ")
-		if err != nil {
-			log.Fatalf("error while prettifying, message = %s, err = %v", message, err)
+		m := jsonpb.Marshaler{Indent: "  "}
+		if err := m.Marshal(os.Stdout, msg); err != nil {
```

Comment:
This seems to combine the end of an object and the beginning of another object on the same line:
```
❯ psql -U postgres -h localhost -d central_data -t \
  -c 'SELECT serialized FROM integration_healths' | \
  go run ./tools/deserialize-proto --type storage.IntegrationHealth
{
  "id": "10d3b4dc-8295-41bc-bb50-6da5484cdb1a",
  "name": "Public DockerHub",
  "type": "IMAGE_INTEGRATION",
  "lastTimestamp": "2024-02-15T00:59:29.769484369Z"
}{ # <- This might be an issue
  "id": "c6a1a26d-8947-4cb0-a50d-a018856f9390",
  "name": "Public Kubernetes GCR (deprecated)",
  "type": "IMAGE_INTEGRATION",
  "lastTimestamp": "2024-02-15T00:59:29.770092380Z"
}%
```